CHANGELOG.md

```diff
@@ -27,14 +27,20 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/CDRH/orchid/compare/v3.1.1...dev) - unreleased
 
+### Fixed
+- displays flash message if API is down, instead of pages throwing errors
+
 ### Changed
 - updates jquery 1.x to 3.x
 - rails g setup is now rails g orchid_setup
+- check_response method added to items_controller
 
 ### Migration
 - modify existing app's `application.js`
   - change `//= require jquery` to `//= require jquery3`
   - apps will continue to function with older jquery if not modified
+  - add `errors.api` and `search.results.error` to locale files
```

Comment:
:raised_hands: These migration notes will be so handy for updating sites with this newer version